### ISSUE TITLE
E-Mail Template, Asset isn't Precompiled.

### ISSUE THREAD

**Issue by jchapin** [capitaltechnologyservices/prelaunchr_for_ni#25]
```
2016-02-24T15:00:02.978701+00:00 app[web.1]: Command :: identify -format '%wx%h,%[exif:orientation]' '/tmp/f90f8e461fe913ca34c51a84849905a120160224-9-ruoqmv.png[0]' 2>/dev/null
2016-02-24T15:00:03.155331+00:00 app[web.1]:   Rendered user_mailer/signup_email.html.erb (1.2ms)
2016-02-24T15:00:03.161333+00:00 app[web.1]: Completed 500 Internal Server Error in 325ms
2016-02-24T15:00:03.128130+00:00 app[web.1]: Command :: file -b --mime '/tmp/6549df7de51f49f98745ccd49020905d20160224-9-aw4phz.png'
2016-02-24T15:00:03.162900+00:00 app[web.1]: ActionView::Template::Error (fb.png isn't precompiled):
2016-02-24T15:00:03.162903+00:00 app[web.1]:   app/controllers/users_controller.rb:34:in `create'
2016-02-24T15:00:03.162904+00:00 app[web.1]: 
2016-02-24T15:00:03.162897+00:00 app[web.1]: 
2016-02-24T15:00:03.160007+00:00 app[web.1]:   Rendered user_mailer/signup_email.html (4.4ms)
2016-02-24T15:00:03.162902+00:00 app[web.1]:   app/mailers/user_mailer.rb:7:in `signup_email'
2016-02-24T15:00:03.162903+00:00 app[web.1]:   app/models/user.rb:58:in `send_welcome_email'
2016-02-24T15:00:03.162901+00:00 app[web.1]:   app/views/user_mailer/signup_email.html:626:in `_app_views_user_mailer_signup_email_html___2480336811173508698_70251446507860'
```

**Comment by jchapin** [comment]
After submitting my contact information and receipt.

ActionView::Template::Error (fb.png isn't precompiled)

**Comment by jchapin** [comment]
There are two template files:

```
peaty:user_mailer jchapin$ ls -l
total 96
-rwxr-xr-x@ 1 jchapin  staff  24100 Feb 24 10:06 signup_email.html
-rwxr-xr-x  1 jchapin  staff  23792 Feb 24 09:45 signup_email.html.erb
```

**Comment by jchapin** [comment]
E-Mail went through after removing the signup_email.html template.

![screen shot 2016-02-24 at 10 14 47 am](https://cloud.githubusercontent.com/assets/116142/13289874/8decf106-dadf-11e5-85cc-b48db71e175b.png)

<img width="838" alt="screen shot 2016-02-24 at 10 15 20 am" src="https://cloud.githubusercontent.com/assets/116142/13289876/8df74a34-dadf-11e5-823b-c4682c9a2975.png">

![screen shot 2016-02-24 at 10 14 43 am](https://cloud.githubusercontent.com/assets/116142/13289875/8def8ef2-dadf-11e5-81e9-81a300c997b5.png)

**Comment by jchapin** [comment]
@capitaltechnologyservices/ruby-developers @RanSolo @cbradshaw, ready for review. Was the signup_email.html template more recent / updated than the content in the html.erb template? I have retained a copy in case that's what needs to be used. We may also have a couple of broken images on the e-mail template.

**Comment by cbradshaw** [comment]
Great. I was using signup_email.html to work from as it was easier to preview in a browser. Left it in there just for reference but should've removed I suppose. 

I'm going to send client the link shortly so he can get us any feedback prior to launch

**Comment by cbradshaw** [comment]
Actually now that I'm testing the email being sent is the template that I worked from so has completely unrelated content from another client. Is it possible to use the signup_email.html rather than the .erb file?

**Comment by jchapin** [comment]
Fixed by #32
